SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -301,15 +312,11 @@ - (void)willTransitionFromStateRegisteredToStateConfigured {
     // before disconnecting Secondary Transport, stop running services
     SDLLogD(@"Stopping services on secondary transport");
     [self sdl_handleTransportUpdateWithPrimaryAvailable:YES secondaryAvailable:NO];
-
-    [self sdl_disconnectSecondaryTransport];
 }
 
 - (void)willTransitionFromStateRegisteredToStateReconnecting {
     SDLLogD(@"Stopping services on secondary transport");
     [self sdl_handleTransportUpdateWithPrimaryAvailable:YES secondaryAvailable:NO];
-
-    [self sdl_disconnectSecondaryTransport];
```

Comment:
The streaming media manager will handle shutting down the secondary transport if necessary when `sdl_handleTransportUpdateWithPrimaryAvailable:secondaryAvailable` is  called. Otherwise the secondary transport may be shutdown before the end video and audio services are sent.